src/settings-ui/Microsoft.PowerToys.Settings.UI/Views/PowerLauncherPage.xaml

```diff
@@ -176,6 +176,158 @@
                     <TextBlock x:Uid="Windows_Color_Settings" />
                 </HyperlinkButton>
             </StackPanel>
+
+            <TextBlock x:Uid="PowerLauncher_Plugins"
+                       Style="{StaticResource SettingsGroupTitleStyle}"
+                       Foreground="{x:Bind Mode=OneWay, Path=ViewModel.EnablePowerLauncher, Converter={StaticResource ModuleEnabledToForegroundConverter}}"/>
+
+            <TextBlock x:Uid="Run_AllPluginsDisabled"
+                       FontSize="12"
+                       Foreground="{ThemeResource SystemControlErrorTextForegroundBrush}"
+                       Visibility="{x:Bind ViewModel.AllPluginsDisabled, Mode=OneWay, Converter={StaticResource BoolToVisibilityConverter}}"
+                       TextWrapping="Wrap"
+                       Margin="{StaticResource SmallTopMargin}"/>
```

Comment:
Here we need a condition to change the foreground into a lighter red when the module is disabled.
_See my [#9653 comment](https://github.com/microsoft/PowerToys/issues/9653#issuecomment-785066582) too._

![image](https://user-images.githubusercontent.com/61519853/109018149-27e04000-76b8-11eb-853c-68382154abc8.png)

Reply:
Now, we do not show that warning when the module is disabled.

Reply:
@mlmykhailopylyp
Can you please check after merge #9880 if the per plugin warnings are hidden too, if Run is disabled. (I have no time at the moment.)

I think if we hide one warning on disabling PT Run, we should hide all for an consistent ux.